src/styles/blocks/card.scss

```
@@ -0,0 +1,84 @@
+* {
+  box-sizing: border-box;
+}
+
+.product-card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  &__img {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin: 0 auto 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $title-color;
+  }
+
+  &__text {
+    margin: 0 0 16px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $text-color;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__review-text {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $title-color;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__price-title {
+    font-size: 12px;
+    line-height: 18px;
+    color: $text-color;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: $title-color;
+  }
```

Use scss nesting, fix it everywhere
```suggestion
  &__price {
    display: flex;
    justify-content: space-between;
    margin-bottom: 16px;
    
    &-title {
      // write styles here
    }
    
    &-value {
      // write styles here
    }
  }
```